src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -8,16 +8,29 @@
 /**
  * Clears the address book.
  */
-public class ClearCommand extends Command {
+public class ClearCommand extends UndoableCommand {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "HealthSync has been cleared!";
+    public static final String MESSAGE_UNDO_SUCCESS = "Undoing the clearing of HealthSync data.";
```

Comment:
Perhaps it could be named `MESSAGE_UNDO_CLEAR_SUCCESS` to be aligned with the variable name in `AddCommand` and `DeleteCommand`.

Reply:
Referring to line 15 here.